content/2017-01-08_pyladies-advanced.md

```diff
@@ -0,0 +1,20 @@
+Title: Pyladies Advanced: Srazy pro programováníchtivé dívky 
+Date: 2017-01-08 10:49:00 
+Author: Lenka 'Zouví' Matějová 
+Twitter: Zoellenka
+ Gravatar: zoelles@gmail.com
+About: Zou je holka, co se učí Python. Má stránky [Zoelles](http://zoelles.tumblr.com/), leze po skalách, chodí po slackline a miluje akrojógu.
+
+Od července do půle října 2016 probíhaly různě po Praze v kavárnách, posléze, díky Oskarovi,  v coworkingové zasedačce [Usertech](https://usertechnologies.com/) na Smíchově. Což je super prostor! Děkujeme!
```

Comment:
Trochu mi v té první větě chybí podmět - kdo/co probíhaly. Já vím, že to je v nadpisu, ale stejně.